runtime/parachains/src/disputes.rs

```diff
@@ -831,32 +782,7 @@ impl<T: Config> Pallet<T> {
 				dispute.concluded_at = Some(now);
 				<Disputes<T>>::insert(session_index, candidate_hash, &dispute);
 
-				if <Included<T>>::contains_key(&session_index, &candidate_hash) {
-					// Local disputes don't count towards spam.
-
-					weight += T::DbWeight::get().reads_writes(1, 1);
-					continue
-				}
-
-				// mildly punish all validators involved. they've failed to make
-				// data available to others, so this is most likely spam.
-				SpamSlots::<T>::mutate(session_index, |spam_slots| {
-					let spam_slots = match spam_slots {
-						Some(ref mut s) => s,
-						None => return,
-					};
-
-					// also reduce spam slots for all validators involved, if the dispute was unconfirmed.
-					// this does open us up to more spam, but only for validators who are willing
-					// to be punished more.
-					//
-					// it would be unexpected for any change here to occur when the dispute has not concluded
-					// in time, as a dispute guaranteed to have at least one honest participant should
-					// conclude quickly.
-					let _participating = decrement_spam(spam_slots, &dispute);
-				});
-
-				weight += T::DbWeight::get().reads_writes(2, 2);
+				weight += T::DbWeight::get().reads_writes(1, 1);
```

Comment:
I might be missing something but I don't see where we do a read here. Before it was `1, 1` up above, but we were also calling `contains_key`, which we don't now.

Reply:
Yes, I think you are right. Weight should be `(0, 1)`.
Thank you!